engine/runtime/src/main/scala/org/enso/compiler/context/SuggestionBuilder.scala

```diff
@@ -108,6 +110,7 @@ final class SuggestionBuilder[A: IndexedSource](
                   _
                 ) if !m.isStaticWrapperForInstanceMethod =>
             val typeSignature = ir.getMetadata(TypeSignatures)
+            val annotations   = ir.getMetadata(GenericAnnotations)
```

Comment:
The suggestion build takes `GenericAnnotations` from `IR.Module.Scope.Definition.Method`'s `IR` and encodes them in the suggestion database, right?

Reply:
Correct